### PR TITLE
JOSS paper folder + update citations

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

File: docs/Project.toml [hunk 1]
```diff
@@ -3,6 +3,7 @@ CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
 FourierFlows = "2aec4490-903f-5c70-9b11-9bed06a700e1"
+GeophysicalFlows = "44ee3b1c-bc02-53fa-8355-8e347616e15e"
```
Comment (glwagner) [inline docs/Project.toml]: 👁️

**Review by glwagner — APPROVED** [review]

File: docs/Project.toml (hunk 1 above)
Comment (navidcy) [reply docs/Project.toml]: that was only there for 40secs! amazing you noticed it!

Comment (glwagner) [reply docs/Project.toml]: 👀

Comment (navidcy) [reply docs/Project.toml]: these eyes are a bit creepy
#1984

Comment (glwagner) [reply docs/Project.toml]: 🙄

**Pull request merged**